### PR TITLE
feat(medical-personnel): added extra levels

### PR DESCRIPTION
**Pull request by dkoutsonikolis**
## Description

Add more levels in the Medical Personnel DB table.

## Checklist

- [x] I have reviewed my code and checked that there are no unrelated changes in this pull request (old patches, accidental config files, etc)
- [ ] I have created at least one test case for the changes I have made
- [x] I have updated the documentation (**SwaggerHub**, **README.md**, **Wiki**, etc.) for the changes I have made

### CODE REVIEW

File: tmh_registry/users/models.py [hunk 1]
```diff
@@ -17,12 +26,12 @@ class MedicalPersonnel(models.Model):
...
+        choices=LEVEL_CHOICES,
+        default=LEAD_SURGEON,
     )
 
     class Meta:
         verbose_name_plural = "Medical Personnel"
 
     def __str__(self):
-        return self.user.username
+        return f"({self.level}) {self.user.username}"
```
Comment (dpapaspyros) [inline tmh_registry/users/models.py:37]: @dkoutsonikolis What about `self.get_level_display()` instead?

Comment (dkoutsonikolis) [reply tmh_registry/users/models.py]: I changed it - not sure if there is a better way.

Comment (dkoutsonikolis) [reply tmh_registry/users/models.py]: Never mind, alll good. :)

**Review by diogenis-si — APPROVED** [review]

**Review by dpapaspyros — APPROVED** [review]
LGTM!!

**Pull request merged**